src/commands/config/getContexts.ts

```diff
@@ -0,0 +1,29 @@
+import { Arguments, createCommand, createTable } from "../../util";
+
+function run(argv: Arguments) {
+  const config = argv.context.skycliConfig;
+  const currentContext = argv.context.currentContext;
+  const contexts = config?.contexts ?? [];
+  const table = createTable({
+    head: ["CURRENT", "CLUSTER", "USER"],
```

Comment:
The most important information is not included here. We should print `CURRENT`, `CONTEXT`, `CLUSTER` and `USER`, where `CONTEXT` is the name of the context. So that the developer can `get-contexts` and then `use-context`.